net-p2p/retroshare/retroshare-0.6.7.ebuild

```diff
@@ -0,0 +1,138 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic desktop qmake-utils xdg
+
+DESCRIPTION="Friend to Friend secure communication and sharing application"
+HOMEPAGE="https://retroshare.cc"
+SRC_URI="https://download.opensuse.org/repositories/network:/retroshare/Debian_Testing/retroshare-common_${PV}.orig.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/RetroShare"
+# NOTE: GitHub releases/archive is impractical to build so we use the OBS repo
+
+LICENSE="AGPL-3 Apache-2.0 CC-BY-SA-4.0 GPL-2 GPL-3 LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="keyring cli +gui +jsonapi libupnp +miniupnp +service +sqlcipher plugins"
+
+REQUIRED_USE="
+	|| ( gui service )
+	?? ( libupnp miniupnp )
+	plugins? ( gui )
+	service? ( || ( cli jsonapi ) )"
+
+RDEPEND="
+	app-arch/bzip2
+	dev-libs/openssl:0=
+	sys-libs/zlib
+	keyring? ( app-crypt/libsecret )
+	gui? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qtmultimedia:5
+		dev-qt/qtnetwork:5
+		dev-qt/qtprintsupport:5
+		dev-qt/qtscript:5
+		dev-qt/qtxml:5
+		dev-qt/qtwidgets:5
+		dev-qt/qtx11extras:5
+		x11-libs/libX11
+		x11-libs/libXScrnSaver
+	)
+	libupnp? ( net-libs/libupnp:= )
+	miniupnp? ( net-libs/miniupnpc:= )
+	sqlcipher? ( dev-db/sqlcipher )
+	!sqlcipher? ( dev-db/sqlite:3 )
+	plugins? (
+		media-libs/speex
+		<media-video/ffmpeg-5
```

Comment:
```suggestion
	plugins? (
		media-libs/speex
		media-libs/speexdsp
		<media-video/ffmpeg-5
```

Reply:
Makes sense.
Resolving this since PR's closed?

Reply:
Probably forgot when I started looking into qtscript dep. I'll add the dep.

Reply:
A mistake then.

Glad that it;s fixed!